From eef02d896e38ab4fbbe62eaa7c6048016a1b11f7 Mon Sep 17 00:00:00 2001
From: Pavel Vasilyev <pavel@pavlinux.ru>
Date: Thu, 29 Oct 2009 17:16:00 +0100
Patch-mainline: 2.6.33-rc1
References: fate#310916
Git-commit: 4f65ae36f0291ef97b7d4de2f59b2e68f3c8420b
Subject: [PATCH 2352/2588] agp/amd64: Remove GART dependency on AGP_AMD64

The GART IOMMU code has no strong dependency to the AMD64
AGP code. So the automatic selection of AGP_AMD64 for GART
can be removed.

Cc: Dave Jones <davej@redhat.com>
Signed-off-by: Pavel Vasilyev <pavel@pavlinux.ru>
Signed-off-by: Joerg Roedel <joerg.roedel@amd.com>
(cherry picked from commit 4f65ae36f0291ef97b7d4de2f59b2e68f3c8420b)

Signed-off-by: Takashi Iwai <tiwai@suse.de>
---
 drivers/char/agp/Kconfig |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/drivers/char/agp/Kconfig b/drivers/char/agp/Kconfig
index 70d56b6..2b64570 100644
--- a/drivers/char/agp/Kconfig
+++ b/drivers/char/agp/Kconfig
@@ -56,7 +56,7 @@ config AGP_AMD
 	  X on AMD Irongate, 761, and 762 chipsets.
 
 config AGP_AMD64
-	tristate "AMD Opteron/Athlon64 on-CPU GART support" if !GART_IOMMU
+	tristate "AMD Opteron/Athlon64 on-CPU GART support"
 	depends on AGP && X86 && K8_NB
 	default y if GART_IOMMU
 	help
-- 
1.7.6

